src/send/hooks.ts

```diff
@@ -18,6 +17,20 @@ import { useSelector } from 'src/redux/hooks'
 import { isValidAddress } from 'src/utils/address'
 import { parsePhoneNumber } from 'src/utils/phoneNumbers'
 
+// Ref: https://github.com/valora-inc/resolve-kit/blob/f84005ea0b522fb6ae40e10ab53d07cf8ef823ef/src/types.ts#L3
+export enum ResolutionKind {
+  Address = 'address',
+  Nom = 'nom',
+}
```

Comment:
Can we copy the latest types here? https://github.com/valora-inc/resolve-kit/blob/main/src/types.ts

Reply:
@jeanregisser latest one doesn't have the `Nom` value which is one of the two actually used. And if I keep the existing ones - knip will fail as those are unused.

Reply:
Hum ok, so the wallet needs to be updated.
Cause we don't resolve nom names anymore then if it's been removed from `@valora/resolve-kit`. It's deployed in a Cloud Function.

We can do that separately.